crates/astria-sequencer/src/ibc/ics20_transfer.rs

```diff
@@ -430,4 +504,159 @@ mod test {
         let asset = Denom::from("other_port/source_channel/asset".to_string());
         assert!(!is_prefixed(&source_port, &source_channel, &asset));
     }
+
+    #[tokio::test]
+    async fn execute_ics20_transfer_to_eoa() {
+        let storage = cnidarium::TempStorage::new()
+            .await
+            .expect("failed to create temp storage backing chain state");
```

Comment:
This message here is wrong as well. IMO the intent here is clear - you just want a temp storage. Feel free to `unwrap` here.